core/src/Cuda/Kokkos_Cuda_Task.hpp

```diff
@@ -468,10 +465,12 @@ class TaskQueueSpecializationConstrained<
   static void execute(scheduler_type const& scheduler) {
     const int shared_per_warp = 2048;
     const int warps_per_block = 4;
+    const Kokkos::Cuda exec   = Cuda();  // FIXME_CUDA_MULTIPLE_DEVICES
+    const auto& impl_instance = exec.impl_internal_space_instance();
     const int multi_processor_count =
         // FIXME not sure why this didn't work
-        // scheduler.get_execution_space().cuda_device_prop().multiProcessorCount;
-        CudaInternal::singleton().m_deviceProp.multiProcessorCount;
+        // exec.cuda_device_prop().multiProcessorCount;
```

Comment:
The `scheduler_type` passed here doesn't have a `get_execution_space()` member function.